models.py

```diff
@@ -91,6 +114,7 @@ def serialize(self):
         return {
             'collection': self.collection,
             'bookNumber': self.bookNumber,
+            'chapterNumber': str(self.babID),
```

Comment:
We already have this in the language-specific portion. And `babID` differs from `babNumber` and if we call it the same thing (`chapterNumber`) it will lead to confusion. I would rather not expose `babID`.